hrms/api/__init__.py

```diff
@@ -624,3 +625,26 @@ def get_allowed_states_for_workflow(workflow: dict, user_id: str) -> list[str]:
 	return [
 		transition.state for transition in workflow.transitions if transition.allowed in user_roles
 	]
+
+# Notification
+@frappe.whitelist(methods=["GET"])
+def subscribe_push_notification(fcm_token: str):
+	current_user = frappe.session.user
+	data = frappe.db.get_value("User", current_user, ["name"], as_dict=True)
+	username = data.name
+	success, message = FrappeNotification().add_token(username, fcm_token)
+	return {
+		"success": success,
+		"message": message
+	}
+
+@frappe.whitelist(methods=["GET"])
+def unsubscribe_push_notification(fcm_token: str):
+	current_user = frappe.session.user
+	data = frappe.db.get_value("User", current_user, ["name"], as_dict=True)
+	username = data.name
+	success, message = FrappeNotification().remove_token(username, fcm_token)
```

Comment:
```suggestion
	success, message = FrappeNotification().remove_token(frappe.session.name, fcm_token)
```

frappe.session.user stores the name of the user so can be used directly?

Reply:
Yeah we can use that

Reply:
done !